bigquery/google/cloud/bigquery/bqdb/__init__.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Google BigQuery implementation of the Database API Specification v2.0.
+
+This module implements the `Python Database API Specification v2.0 (DB-API)`_
+for Google BigQuery.
+
+.. _Python Database API Specification v2.0 (DB-API): https://www.python.org/dev/peps/pep-0249/
+
+.. warning::
+   The ``bqdb`` module is **alpha**. The implementation is not complete. It
+   might be changed in backward-incompatible ways and is not subject to any SLA
+   or deprecation policy.
+"""
+
+from google.cloud.bigquery.bqdb.connection import connect
+from google.cloud.bigquery.bqdb.connection import Connection
+from google.cloud.bigquery.bqdb.cursor import Cursor
+from google.cloud.bigquery.bqdb.exceptions import Warning
+from google.cloud.bigquery.bqdb.exceptions import Error
+from google.cloud.bigquery.bqdb.exceptions import InterfaceError
+from google.cloud.bigquery.bqdb.exceptions import DatabaseError
+from google.cloud.bigquery.bqdb.exceptions import DataError
+from google.cloud.bigquery.bqdb.exceptions import OperationalError
+from google.cloud.bigquery.bqdb.exceptions import IntegrityError
+from google.cloud.bigquery.bqdb.exceptions import InternalError
+from google.cloud.bigquery.bqdb.exceptions import ProgrammingError
+from google.cloud.bigquery.bqdb.exceptions import NotSupportedError
+
+
+apilevel = "2.0"
+
+# Threads may share the module, but not connections.
```

Comment:
I'm guessing if we switch to a threadsafe transport then we can use `threadsafety = 2`?